src/components/CurrencyInput/CurrencyInput.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from 'react-emotion';
+import { withTheme } from 'emotion-theming';
+
+import IconInput from '../IconInput';
+import HtmlElement from '../HtmlElement';
+import { childrenPropType, themePropType } from '../../util/shared-prop-types';
+
+const currencyBaseStyles = ({ theme }) => css`
+  color: ${theme.colors.b700};
+`;
+
+const currencyStyle = ({ theme, className }) =>
+  cx(currencyBaseStyles({ theme }), className);
+
+/**
+ * SearchInput component for forms.
+ */
+const CurrencyInput = ({
+  children,
+  currencyPosition,
+  currency,
+  theme,
+  ...props
+}) => (
+  <IconInput
+    iconLeft={({ className }) =>
+      currencyPosition === 'left' && (
+        <HtmlElement className={currencyStyle({ theme, className })}>
```

Comment:
What kind of `<HtmlElement>` do you want to render here? You need to pass it a string or function. See the [prop types](https://github.com/sumup/circuit-ui/blob/master/src/components/HtmlElement/HtmlElement.js#L20-L27) for documentation:

```js
HtmlElement.propTypes = {
  /**
   * The HTML element that should be returned. If a function is passed,
   * the function needs to return a string for an HTML element (i.e., 'button').
   * The function is called with all component props except "element" and "children".
   * See the Button component for an example of how to use this.
   */
  element: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
  // ...
};
```

Reply:
I think it makes more sense to just use a `<span>` here. WDYT?

Reply:
Yeah, there is no need to use `HtmlElement` here. You can use a styled `<span>` or `<div>` right away. Then you also don't need the `currencyStyle` and `cx` functions.